inc/taskview.class.php

```diff
@@ -284,6 +284,20 @@ public function showJobLogs()
       });");
     }
 
+    /**
+    * Prepare data before update in database
+    *
+    * @param array $input
+    * @return array
+    */
+    public function prepareInputForUpdate($input)
```

Comment:
Is this normal that this change is made on `PluginGlpiinventoryTaskView`? I may miss something, but I guess that the data that your are trying to check is related to `PluginGlpiinventoryTask`.

Reply:
you're right